savegame.json

```diff
@@ -622,18 +622,8 @@
     },
     {
       "Type": 0,
-      "Name": "CardMask",
-      "URL": "http://cloud-3.steamusercontent.com/ugc/2042997062885048476/D9364C4FF33C4A1D87D5C1B8B7690FB32F05E1E3/"
-    },
-    {
-      "Type": 0,
-      "Name": "SpiritMask",
-      "URL": "http://cloud-3.steamusercontent.com/ugc/2050879298353456785/D66B86DC757A20FFFB766E7734211B7298E2B91C/"
-    },
-    {
-      "Type": 0,
-      "Name": "SpiritMask2",
-      "URL": "http://cloud-3.steamusercontent.com/ugc/2050879298353456583/D2F073D12EE9155AF0687241797E99610212B347/"
+      "Name": "ReminderMask",
+      "URL": "http://cloud-3.steamusercontent.com/ugc/2144334608481559493/F8DA1252B35DC62BE5B9A13CF53FAD7CD5320C21/"
```

Comment:
so what's different about this mask compared to the other 3? or i guess asking differently why are none of those 3 good to use?

Reply:
Those three masks are each the shape of a card/spirit panel, with the hole punched in the correct place. That makes them very easy to use when overlayed directly on an entire card/spirit panel image, as they line up in the right place. But finding the "centre" of the hole in them, to perform the calculations to convert reminder token position to UI image position, is a tricky prospect. The new mask is a square centred on the "centre" of the hole, and makes these calculations work much more nicely.

It was still fairly tricky creating this mask correctly to get the centre in the right place, 'cause the "centre" of a reminder token as obtained by taking the halfway point between the top/bottom and left/right edges is *not* the centre of the object model's coordinate system. To get the centre in the right place required me to build the mask image *based on* the reminder token's object model coordinate system, using Blender. I document the process in the relevant commit message (fc94c258093ac9b5479610b695482d070ffd1b1d); I'm not sure if there's somewhere better to toss that documentation as well, just in case it needs redoing some time in the future.

Reply:
gotcha so instead of the mask being the size of a card or panel, it's basically just the size of the cutout itself. that makes sense, thanks